.github/workflows/trivy.yml

```diff
@@ -0,0 +1,21 @@
+name: Trivy scanner for docker
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
```

Comment:
I would replace this comment with just: schedule to run hourly

Reply:
👍